packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.html

```diff
@@ -1,11 +1,28 @@
 <h1 id="dedicatedCloud_datacenter_backup_header" data-translate="dedicatedCloud_datacenter_backup_title"></h1>
 <div data-ui-view>
     <div class="datacenter-backup">
-        <div data-ovh-alert="{{ ::$ctrl.alertMain }}"></div>
-        <oui-switch data-model="true" data-on-change="$ctrl.deactivate()">
+        <div data-ovh-alert="{{ $ctrl.alertMain }}" id="backupMainAlert"></div>
```

Comment:
The `id` is no longer required. Please take this change from the parent branch `feat/veeam-backup`

Reply:
it will be taken care during rebase/merge